LICENSE.txt

```diff
@@ -1,73 +1,8 @@
-the patches directory is retained by the NeoForge project.
-This is because the patches are partially machine generated
-artifacts, and are changed heavily due to the way NeoForge works.
-Individual attribution within them is impossible.
-
-Consent:
-All contributions to NeoForge must consent to the release of any
-patch content to the NeoForge project.
-
-A note on infectivity:
-The LGPL is chosen specifically so that projects may depend on NeoForge
-features without being infected with its license. That is the 
-purpose of the LGPL. Mods and others using this code via ordinary
-Java mechanics for referencing libraries are specifically not bound
-by NeoForge's license for the Mod code.
-
-
-=== MCP Data ===
-This software includes data from the Minecraft Coder Pack (MCP), with kind permission
-from them. The license to MCP data is not transitive - distribution of this data by
-third parties requires independent licensing from the MCP team. This data is not
-redistributable without permission from the MCP team.
-
-=== Sharing ===
-I grant permission for some parts of FML to be redistributed outside the terms of the LGPL, for the benefit of
-the minecraft modding community. All contributions to these parts should be licensed under the same additional grant.
-
--- Runtime patcher --
-License is granted to redistribute the runtime patcher code (src/main/java/net/minecraftforge/fml/common/patcher
-and subdirectories) under any alternative open source license as classified by the OSI (http://opensource.org/licenses)
-
--- Coremods --
-License is granted to redistribute the Coremod code (/src/main/resources/coremods)
-under any alternative open source license as classified by the OSI (http://opensource.org/licenses)
-
-=========================================================================
-This software includes portions from the Apache Maven project at
-http://maven.apache.org/ specifically the ComparableVersion.java code. It is
-included based on guidelines at
-http://www.softwarefreedom.org/resources/2007/gpl-non-gpl-collaboration.html
-with notices intact. The only change is a non-functional change of package name.
-
-This software contains a partial repackaging of javaxdelta, a BSD licensed program for generating
-binary differences and applying them, sourced from the subversion at http://sourceforge.net/projects/javaxdelta/
-authored by genman, heikok, pivot.
-The only changes are to replace some Trove collection types with standard Java collections, and repackaged.
-
-This software includes the Monocraft font from https://github.com/IdreesInc/Monocraft/ for use in the early loading
-display.
-=========================================================================
-
-
                   GNU LESSER GENERAL PUBLIC LICENSE
                        Version 2.1, February 1999
 
  Copyright (C) 1991, 1999 Free Software Foundation, Inc.
- 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
```

Comment:
Heh. you can tell how old it is because there's a physical address.

Reply:
And now it's a web address! 😄